src/overlays/actors/ovl_En_Tanron5/z_en_tanron5.c

```diff
@@ -28,32 +35,493 @@ const ActorInit En_Tanron5_InitVars = {
+                        sp9C.y = this->actor.floorHeight + (spAC * D_80BE5DD0);
+
+                        Actor_Spawn(&globalCtx->actorCtx, globalCtx, ACTOR_EN_TANRON5, sp9C.x, sp9C.y, sp9C.z,
+                                    Rand_ZeroFloat(0x10000), Rand_ZeroFloat(0x10000), 0, i + 100);
+
+                        if (i < 8) {
+                            func_80BE4A2C(globalCtx->specialEffects, &sp9C, Rand_ZeroFloat(3.0f) + 6.0f);
+                        }
+                    }
+                }
+
+                if (Rand_ZeroOne() < 0.333f) {
+                    phi_v0 = 0x4000;
+                } else if (Rand_ZeroOne() < 0.5f) {
+                    phi_v0 = -0x8000;
+                } else {
+                    phi_v0 = -0x4000;
+                }
+
+                this->actor.shape.rot.y += phi_v0;
+                Actor_PlaySfxAtPos(&this->actor, NA_SE_IT_BIG_BOMB_EXPLOSION);
+                func_800BC848(&this->actor, globalCtx, 4, 4);
+                this->unk_1A0++;
+            } else {
+                Vec3f sp90;
+                ColliderInfo* info = this->collider.info.acHitInfo;
+
+                sp90.x = info->bumper.hitPos.x;
+                sp90.y = info->bumper.hitPos.y;
+                sp90.z = info->bumper.hitPos.z;
+
+                Actor_PlaySfxAtPos(&this->actor, NA_SE_IT_SHIELD_REFLECT_SW);
+                CollisionCheck_SpawnShieldParticlesMetal(globalCtx, &sp90);
+            }
+        }
+    }
+
+    Collider_UpdateCylinder(&this->actor, &this->collider);
+    if (this->unk_148 == object_boss02_DL_006FD0) {
+        this->collider.dim.pos.y = this->actor.floorHeight;
+    }
+
+    if ((this->unk_148 == object_boss02_DL_007D18) || (D_80BE5DD0 < 0.5f)) {
+        CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+    } else {
+        f32 xDiff = player->actor.world.pos.x - this->actor.world.pos.x;
+        f32 yDiff = player->actor.world.pos.z - this->actor.world.pos.z;
+
+        if ((fabsf(xDiff) < 120.0f) && (fabsf(yDiff) < 120.0f)) {
+            if (fabsf(yDiff) < fabsf(xDiff)) {
+                if (xDiff > 0.0f) {
+                    player->actor.prevPos.x = player->actor.world.pos.x = this->actor.world.pos.x + 120.0f;
+                } else {
+                    player->actor.prevPos.x = player->actor.world.pos.x = this->actor.world.pos.x - 120.0f;
+                }
+            } else if (yDiff > 0.0f) {
+                player->actor.prevPos.z = player->actor.world.pos.z = this->actor.world.pos.z + 120.0f;
+            } else {
+                player->actor.prevPos.z = player->actor.world.pos.z = this->actor.world.pos.z - 120.0f;
+            }
+        }
+    }
+
+    CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+}
+
+void func_80BE5818(Actor* thisx, GlobalContext* globalCtx2) {
+    f32 sp6C;
+    s32 i;
+    Vec3f sp5C;
+    EnTanron5* this = THIS;
+    GlobalContext* globalCtx = globalCtx2;
```

Comment:
```suggestion
    GlobalContext* globalCtx = globalCtx2;
    EnTanron5* this = THIS;
    f32 sp6C;
    s32 i;
    Vec3f sp5C;
```
Would this still match?

Reply:
Nope, it doesn't. I had it like this but I have to move them down for stack.